portal/package.json

```diff
@@ -27,6 +27,7 @@
     "start": "node server/index.js",
     "test": "jest --coverage --verbose --config=jest.config.js",
     "test-dev": "jest --verbose --config=jest.config.js",
+    "test-file": "jest --config=jest.config.js --verbose --testMatch",
```

Comment:
Should we document this new command in the README?

Reply:
updated